DEB [20190426-16:37:50.652] thr=1   paramiko.transport: starting thread (client mode): 0x3434c50
DEB [20190426-16:37:50.652] thr=2   paramiko.transport: starting thread (client mode): 0x3434d50
DEB [20190426-16:37:50.653] thr=3   paramiko.transport: starting thread (client mode): 0x418f250
DEB [20190426-16:37:50.653] thr=4   paramiko.transport: starting thread (client mode): 0x418f0f0
DEB [20190426-16:37:50.653] thr=5   paramiko.transport: starting thread (client mode): 0x418f510
DEB [20190426-16:37:50.654] thr=6   paramiko.transport: starting thread (client mode): 0x418f3b0
DEB [20190426-16:37:50.654] thr=1   paramiko.transport: Local version/idstring: SSH-2.0-paramiko_2.4.2
DEB [20190426-16:37:50.654] thr=2   paramiko.transport: Local version/idstring: SSH-2.0-paramiko_2.4.2
DEB [20190426-16:37:50.654] thr=3   paramiko.transport: Local version/idstring: SSH-2.0-paramiko_2.4.2
DEB [20190426-16:37:50.654] thr=4   paramiko.transport: Local version/idstring: SSH-2.0-paramiko_2.4.2
DEB [20190426-16:37:50.655] thr=6   paramiko.transport: Local version/idstring: SSH-2.0-paramiko_2.4.2
DEB [20190426-16:37:50.655] thr=5   paramiko.transport: Local version/idstring: SSH-2.0-paramiko_2.4.2
DEB [20190426-16:37:50.668] thr=1   paramiko.transport: Remote version/idstring: SSH-2.0-OpenSSH_7.4
INF [20190426-16:37:50.668] thr=1   paramiko.transport: Connected (version 2.0, client OpenSSH_7.4)
DEB [20190426-16:37:50.673] thr=1   paramiko.transport: kex algos:['curve25519-sha256', 'curve25519-sha256@libssh.org', 'ecdh-sha2-nistp256', 'ecdh-sha2-nistp384', 'ecdh-sha2-nistp521', 'diffie-hellman-group-exchange-sha256', 'diffie-hellman-group16-sha512', 'diffie-hellman-group18-sha512', 'diffie-hellman-group-exchange-sha1', 'diffie-hellman-group14-sha256', 'diffie-hellman-group14-sha1', 'diffie-hellman-group1-sha1'] server key:['ssh-rsa', 'rsa-sha2-512', 'rsa-sha2-256', 'ecdsa-sha2-nistp256', 'ssh-ed25519'] client encrypt:['chacha20-poly1305@openssh.com', 'aes128-ctr', 'aes192-ctr', 'aes256-ctr', 'aes128-gcm@openssh.com', 'aes256-gcm@openssh.com', 'aes128-cbc', 'aes192-cbc', 'aes256-cbc', 'blowfish-cbc', 'cast128-cbc', '3des-cbc'] server encrypt:['chacha20-poly1305@openssh.com', 'aes128-ctr', 'aes192-ctr', 'aes256-ctr', 'aes128-gcm@openssh.com', 'aes256-gcm@openssh.com', 'aes128-cbc', 'aes192-cbc', 'aes256-cbc', 'blowfish-cbc', 'cast128-cbc', '3des-cbc'] client mac:['umac-64-etm@openssh.com', 'umac-128-etm@openssh.com', 'hmac-sha2-256-etm@openssh.com', 'hmac-sha2-512-etm@openssh.com', 'hmac-sha1-etm@openssh.com', 'umac-64@openssh.com', 'umac-128@openssh.com', 'hmac-sha2-256', 'hmac-sha2-512', 'hmac-sha1'] server mac:['umac-64-etm@openssh.com', 'umac-128-etm@openssh.com', 'hmac-sha2-256-etm@openssh.com', 'hmac-sha2-512-etm@openssh.com', 'hmac-sha1-etm@openssh.com', 'umac-64@openssh.com', 'umac-128@openssh.com', 'hmac-sha2-256', 'hmac-sha2-512', 'hmac-sha1'] client compress:['none', 'zlib@openssh.com'] server compress:['none', 'zlib@openssh.com'] client lang:[''] server lang:[''] kex follows?False
DEB [20190426-16:37:50.673] thr=1   paramiko.transport: Kex agreed: ecdh-sha2-nistp256
DEB [20190426-16:37:50.673] thr=1   paramiko.transport: HostKey agreed: ssh-ed25519
DEB [20190426-16:37:50.673] thr=1   paramiko.transport: Cipher agreed: aes128-ctr
DEB [20190426-16:37:50.673] thr=1   paramiko.transport: MAC agreed: hmac-sha2-256
DEB [20190426-16:37:50.673] thr=1   paramiko.transport: Compression agreed: none
DEB [20190426-16:37:50.681] thr=2   paramiko.transport: Remote version/idstring: SSH-2.0-OpenSSH_7.9p1 Debian-9
INF [20190426-16:37:50.681] thr=2   paramiko.transport: Connected (version 2.0, client OpenSSH_7.9p1)
DEB [20190426-16:37:50.683] thr=2   paramiko.transport: kex algos:['curve25519-sha256', 'curve25519-sha256@libssh.org', 'ecdh-sha2-nistp256', 'ecdh-sha2-nistp384', 'ecdh-sha2-nistp521', 'diffie-hellman-group-exchange-sha256', 'diffie-hellman-group16-sha512', 'diffie-hellman-group18-sha512', 'diffie-hellman-group14-sha256', 'diffie-hellman-group14-sha1'] server key:['rsa-sha2-512', 'rsa-sha2-256', 'ssh-rsa', 'ecdsa-sha2-nistp256', 'ssh-ed25519'] client encrypt:['chacha20-poly1305@openssh.com', 'aes128-ctr', 'aes192-ctr', 'aes256-ctr', 'aes128-gcm@openssh.com', 'aes256-gcm@openssh.com'] server encrypt:['chacha20-poly1305@openssh.com', 'aes128-ctr', 'aes192-ctr', 'aes256-ctr', 'aes128-gcm@openssh.com', 'aes256-gcm@openssh.com'] client mac:['umac-64-etm@openssh.com', 'umac-128-etm@openssh.com', 'hmac-sha2-256-etm@openssh.com', 'hmac-sha2-512-etm@openssh.com', 'hmac-sha1-etm@openssh.com', 'umac-64@openssh.com', 'umac-128@openssh.com', 'hmac-sha2-256', 'hmac-sha2-512', 'hmac-sha1'] server mac:['umac-64-etm@openssh.com', 'umac-128-etm@openssh.com', 'hmac-sha2-256-etm@openssh.com', 'hmac-sha2-512-etm@openssh.com', 'hmac-sha1-etm@openssh.com', 'umac-64@openssh.com', 'umac-128@openssh.com', 'hmac-sha2-256', 'hmac-sha2-512', 'hmac-sha1'] client compress:['none', 'zlib@openssh.com'] server compress:['none', 'zlib@openssh.com'] client lang:[''] server lang:[''] kex follows?False
DEB [20190426-16:37:50.684] thr=2   paramiko.transport: Kex agreed: ecdh-sha2-nistp256
DEB [20190426-16:37:50.684] thr=2   paramiko.transport: HostKey agreed: ssh-ed25519
DEB [20190426-16:37:50.684] thr=2   paramiko.transport: Cipher agreed: aes128-ctr
DEB [20190426-16:37:50.684] thr=2   paramiko.transport: MAC agreed: hmac-sha2-256
DEB [20190426-16:37:50.684] thr=2   paramiko.transport: Compression agreed: none
DEB [20190426-16:37:50.732] thr=1   paramiko.transport: kex engine KexNistp256 specified hash_algo <built-in function openssl_sha256>
DEB [20190426-16:37:50.732] thr=1   paramiko.transport: Switch to new keys ...
DEB [20190426-16:37:50.733] thr=2   paramiko.transport: kex engine KexNistp256 specified hash_algo <built-in function openssl_sha256>
DEB [20190426-16:37:50.733] thr=7   paramiko.transport: Adding ssh-ed25519 host key for 10.49.20.152: b'd92fcb1a9255b6eb6daebd0435cf848f'
DEB [20190426-16:37:50.733] thr=2   paramiko.transport: Switch to new keys ...
DEB [20190426-16:37:50.734] thr=8   paramiko.transport: Adding ssh-ed25519 host key for 10.49.20.156: b'110c4253d5eedecae1ba24eafbbe2ba9'
DEB [20190426-16:37:50.773] thr=1   paramiko.transport: userauth is OK
DEB [20190426-16:37:50.776] thr=2   paramiko.transport: userauth is OK
INF [20190426-16:37:50.803] thr=2   paramiko.transport: Authentication (password) successful!
DEB [20190426-16:37:50.803] thr=2   paramiko.transport: EOF in transport thread
INF [20190426-16:37:51.744] thr=1   paramiko.transport: Authentication (password) successful!
DEB [20190426-16:37:51.744] thr=1   paramiko.transport: EOF in transport thread
ERR [20190426-16:38:05.699] thr=3   paramiko.transport: Exception: Error reading SSH protocol banner
ERR [20190426-16:38:05.699] thr=4   paramiko.transport: Exception: Error reading SSH protocol banner
ERR [20190426-16:38:05.699] thr=5   paramiko.transport: Exception: Error reading SSH protocol banner
ERR [20190426-16:38:05.699] thr=6   paramiko.transport: Exception: Error reading SSH protocol banner
ERR [20190426-16:38:05.709] thr=4   paramiko.transport: Traceback (most recent call last):
ERR [20190426-16:38:05.709] thr=4   paramiko.transport:   File "C:\Users\pthumbur\AppData\Local\Programs\Python\Python37-32\lib\site-packages\paramiko\transport.py", line 2138, in _check_banner
ERR [20190426-16:38:05.709] thr=4   paramiko.transport:     buf = self.packetizer.readline(timeout)
ERR [20190426-16:38:05.710] thr=4   paramiko.transport:   File "C:\Users\pthumbur\AppData\Local\Programs\Python\Python37-32\lib\site-packages\paramiko\packet.py", line 367, in readline
ERR [20190426-16:38:05.710] thr=6   paramiko.transport: Traceback (most recent call last):
ERR [20190426-16:38:05.710] thr=4   paramiko.transport:     buf += self._read_timeout(timeout)
ERR [20190426-16:38:05.711] thr=3   paramiko.transport: Traceback (most recent call last):
ERR [20190426-16:38:05.711] thr=6   paramiko.transport:   File "C:\Users\pthumbur\AppData\Local\Programs\Python\Python37-32\lib\site-packages\paramiko\transport.py", line 2138, in _check_banner
ERR [20190426-16:38:05.712] thr=5   paramiko.transport: Traceback (most recent call last):
ERR [20190426-16:38:05.712] thr=4   paramiko.transport:   File "C:\Users\pthumbur\AppData\Local\Programs\Python\Python37-32\lib\site-packages\paramiko\packet.py", line 576, in _read_timeout
ERR [20190426-16:38:05.712] thr=3   paramiko.transport:   File "C:\Users\pthumbur\AppData\Local\Programs\Python\Python37-32\lib\site-packages\paramiko\transport.py", line 2138, in _check_banner
ERR [20190426-16:38:05.712] thr=6   paramiko.transport:     buf = self.packetizer.readline(timeout)
ERR [20190426-16:38:05.713] thr=5   paramiko.transport:   File "C:\Users\pthumbur\AppData\Local\Programs\Python\Python37-32\lib\site-packages\paramiko\transport.py", line 2138, in _check_banner
ERR [20190426-16:38:05.713] thr=4   paramiko.transport:     raise socket.timeout()
ERR [20190426-16:38:05.713] thr=3   paramiko.transport:     buf = self.packetizer.readline(timeout)
ERR [20190426-16:38:05.713] thr=6   paramiko.transport:   File "C:\Users\pthumbur\AppData\Local\Programs\Python\Python37-32\lib\site-packages\paramiko\packet.py", line 367, in readline
ERR [20190426-16:38:05.714] thr=5   paramiko.transport:     buf = self.packetizer.readline(timeout)
ERR [20190426-16:38:05.714] thr=4   paramiko.transport: socket.timeout
ERR [20190426-16:38:05.714] thr=3   paramiko.transport:   File "C:\Users\pthumbur\AppData\Local\Programs\Python\Python37-32\lib\site-packages\paramiko\packet.py", line 367, in readline
ERR [20190426-16:38:05.714] thr=6   paramiko.transport:     buf += self._read_timeout(timeout)
ERR [20190426-16:38:05.714] thr=5   paramiko.transport:   File "C:\Users\pthumbur\AppData\Local\Programs\Python\Python37-32\lib\site-packages\paramiko\packet.py", line 367, in readline
ERR [20190426-16:38:05.715] thr=4   paramiko.transport: 
ERR [20190426-16:38:05.715] thr=3   paramiko.transport:     buf += self._read_timeout(timeout)
ERR [20190426-16:38:05.715] thr=6   paramiko.transport:   File "C:\Users\pthumbur\AppData\Local\Programs\Python\Python37-32\lib\site-packages\paramiko\packet.py", line 576, in _read_timeout
ERR [20190426-16:38:05.715] thr=5   paramiko.transport:     buf += self._read_timeout(timeout)
ERR [20190426-16:38:05.716] thr=4   paramiko.transport: During handling of the above exception, another exception occurred:
ERR [20190426-16:38:05.716] thr=3   paramiko.transport:   File "C:\Users\pthumbur\AppData\Local\Programs\Python\Python37-32\lib\site-packages\paramiko\packet.py", line 576, in _read_timeout
ERR [20190426-16:38:05.716] thr=6   paramiko.transport:     raise socket.timeout()
ERR [20190426-16:38:05.716] thr=5   paramiko.transport:   File "C:\Users\pthumbur\AppData\Local\Programs\Python\Python37-32\lib\site-packages\paramiko\packet.py", line 576, in _read_timeout
ERR [20190426-16:38:05.716] thr=4   paramiko.transport: 
ERR [20190426-16:38:05.717] thr=3   paramiko.transport:     raise socket.timeout()
ERR [20190426-16:38:05.717] thr=6   paramiko.transport: socket.timeout
ERR [20190426-16:38:05.717] thr=5   paramiko.transport:     raise socket.timeout()
ERR [20190426-16:38:05.717] thr=4   paramiko.transport: Traceback (most recent call last):
ERR [20190426-16:38:05.718] thr=3   paramiko.transport: socket.timeout
ERR [20190426-16:38:05.718] thr=6   paramiko.transport: 
ERR [20190426-16:38:05.718] thr=5   paramiko.transport: socket.timeout
ERR [20190426-16:38:05.718] thr=4   paramiko.transport:   File "C:\Users\pthumbur\AppData\Local\Programs\Python\Python37-32\lib\site-packages\paramiko\transport.py", line 1966, in run
ERR [20190426-16:38:05.718] thr=3   paramiko.transport: 
ERR [20190426-16:38:05.719] thr=6   paramiko.transport: During handling of the above exception, another exception occurred:
ERR [20190426-16:38:05.719] thr=5   paramiko.transport: 
ERR [20190426-16:38:05.719] thr=4   paramiko.transport:     self._check_banner()
ERR [20190426-16:38:05.719] thr=3   paramiko.transport: During handling of the above exception, another exception occurred:
ERR [20190426-16:38:05.719] thr=6   paramiko.transport: 
ERR [20190426-16:38:05.720] thr=5   paramiko.transport: During handling of the above exception, another exception occurred:
ERR [20190426-16:38:05.720] thr=4   paramiko.transport:   File "C:\Users\pthumbur\AppData\Local\Programs\Python\Python37-32\lib\site-packages\paramiko\transport.py", line 2143, in _check_banner
ERR [20190426-16:38:05.720] thr=3   paramiko.transport: 
ERR [20190426-16:38:05.720] thr=6   paramiko.transport: Traceback (most recent call last):
ERR [20190426-16:38:05.721] thr=5   paramiko.transport: 
ERR [20190426-16:38:05.721] thr=4   paramiko.transport:     "Error reading SSH protocol banner" + str(e)
ERR [20190426-16:38:05.721] thr=3   paramiko.transport: Traceback (most recent call last):
ERR [20190426-16:38:05.721] thr=6   paramiko.transport:   File "C:\Users\pthumbur\AppData\Local\Programs\Python\Python37-32\lib\site-packages\paramiko\transport.py", line 1966, in run
ERR [20190426-16:38:05.721] thr=5   paramiko.transport: Traceback (most recent call last):
ERR [20190426-16:38:05.722] thr=4   paramiko.transport: paramiko.ssh_exception.SSHException: Error reading SSH protocol banner
ERR [20190426-16:38:05.722] thr=3   paramiko.transport:   File "C:\Users\pthumbur\AppData\Local\Programs\Python\Python37-32\lib\site-packages\paramiko\transport.py", line 1966, in run
ERR [20190426-16:38:05.722] thr=6   paramiko.transport:     self._check_banner()
ERR [20190426-16:38:05.722] thr=5   paramiko.transport:   File "C:\Users\pthumbur\AppData\Local\Programs\Python\Python37-32\lib\site-packages\paramiko\transport.py", line 1966, in run
ERR [20190426-16:38:05.723] thr=4   paramiko.transport: 
ERR [20190426-16:38:05.723] thr=3   paramiko.transport:     self._check_banner()
ERR [20190426-16:38:05.723] thr=6   paramiko.transport:   File "C:\Users\pthumbur\AppData\Local\Programs\Python\Python37-32\lib\site-packages\paramiko\transport.py", line 2143, in _check_banner
ERR [20190426-16:38:05.723] thr=5   paramiko.transport:     self._check_banner()
ERR [20190426-16:38:05.724] thr=3   paramiko.transport:   File "C:\Users\pthumbur\AppData\Local\Programs\Python\Python37-32\lib\site-packages\paramiko\transport.py", line 2143, in _check_banner
ERR [20190426-16:38:05.724] thr=6   paramiko.transport:     "Error reading SSH protocol banner" + str(e)
ERR [20190426-16:38:05.724] thr=5   paramiko.transport:   File "C:\Users\pthumbur\AppData\Local\Programs\Python\Python37-32\lib\site-packages\paramiko\transport.py", line 2143, in _check_banner
ERR [20190426-16:38:05.725] thr=3   paramiko.transport:     "Error reading SSH protocol banner" + str(e)
ERR [20190426-16:38:05.725] thr=6   paramiko.transport: paramiko.ssh_exception.SSHException: Error reading SSH protocol banner
ERR [20190426-16:38:05.725] thr=5   paramiko.transport:     "Error reading SSH protocol banner" + str(e)
ERR [20190426-16:38:05.725] thr=3   paramiko.transport: paramiko.ssh_exception.SSHException: Error reading SSH protocol banner
ERR [20190426-16:38:05.725] thr=6   paramiko.transport: 
ERR [20190426-16:38:05.726] thr=5   paramiko.transport: paramiko.ssh_exception.SSHException: Error reading SSH protocol banner
ERR [20190426-16:38:05.726] thr=3   paramiko.transport: 
ERR [20190426-16:38:05.726] thr=5   paramiko.transport: 
